src/main/java/com/onarandombox/MultiverseCore/destination/DestinationFactory.java

```diff
@@ -164,4 +165,8 @@ public boolean registerDestinationType(Class<? extends MVDestination> c, String
         this.teleportCommand.addAdditonalPermission(other);
         return true;
     }
+    
+    public Collection<String> getRegisteredIdentifiers() {
+        return this.destList.keySet();
+    }
```

Comment:
Easy peasy. But since you were talking about javadocs in discord I think something is missing here...

Reply:
Fixed in #2597.